### ISSUE TITLE
Show/hide population line when population name is clicked in the legend

### ISSUE THREAD

**Issue by openjck** [mozilla/metrics-graphics#808]
This is something that some other charting libraries offer ([Chart.js example](http://www.chartjs.org/samples/latest/charts/line/basic.html)).

This feature was [requested in Ensemble](https://github.com/mozilla/ensemble/issues/15). We can add the feature on top of metrics-graphics ourselves, but I thought it wouldn't hurt to also request it upstream. :smile:

**Comment by wlach** [comment]
This seems like a reasonable request!

**Comment by vedantc98** [comment]
@wlach could I take up this issue?

**Comment by wlach** [comment]
@vedantc98 definitely, though note this bug is probably not easy. I'm not that familiar with this code but the basic place to start with would be this example with a legend:

![image](https://user-images.githubusercontent.com/20569/36102005-e9c649ac-0fd8-11e8-80a6-39e0b93824d0.png)

See: https://www.metricsgraphicsjs.org/examples.htm#multilines

Note how the elements in the legend are not currently interactive. Your task would be to make that behave as in the example @openjck provided.
